Quadriga/src/main/java/edu/asu/spring/quadriga/service/publicwebsite/impl/ProjectStats.java

```diff
@@ -0,0 +1,198 @@
+        HashMap<String, ConceptStats> mapStats = new HashMap<String, ConceptStats>();
+
+        for (INetwork n : networks) {
+            ITransformedNetwork transformedNetwork = transformationManager
+                    .getTransformedNetwork(n.getNetworkId());
+            if (transformedNetwork != null) {
+                Map<String, Node> mapNodes = transformedNetwork.getNodes();
+                for (Node node : mapNodes.values()) {
+                    String url = node.getConceptId();
+                    if (mapStats.containsKey(url)) {
+                        mapStats.get(url).incrementCount();
+                    } else {
+                        ConceptStats conceptStats = new ConceptStats(url,
+                                node.getDescription(), node.getLabel(), 1);
+                        mapStats.put(url, conceptStats);
+                    }
+                }
+            }
+        }
+
+        List<IConceptStats> valuesList = new ArrayList<IConceptStats>(
+                mapStats.values());
+
+        return getSortedList(valuesList);
+
+    }
+
+    public List<IConceptStats> getSortedList(List<IConceptStats> csList) {
+        Collections.sort(csList, new Comparator<IConceptStats>() {
+            public int compare(IConceptStats o1, IConceptStats o2) {
+                if (o1.getCount() == o2.getCount())
+                    return 0;
+                return o1.getCount() < o2.getCount() ? 1 : -1;
+            }
+        });
+        return csList;
+    }
+
+    @Override
+    public Map<String, IUserStats> getCountofWorkspace(String projectId)
+            throws QuadrigaStorageException {
+
+        List<IProjectCollaborator> projectCollaboratorList = projectCollaboratorManager
+                .getProjectCollaborators(projectId);
+
+        if (projectCollaboratorList == null)
+            return null;
+
+        Map<String, IUserStats> mapUserWorkspace = new HashMap<String, IUserStats>();
+        // loop through each collaborator
+
+        for (IProjectCollaborator projectCollaborator : projectCollaboratorList) {
+            if (projectCollaborator.getCollaborator() != null) {
+                String username = projectCollaborator.getCollaborator()
+                        .getUserObj().getUserName();
+                UserStats userStats = (UserStats) setUserWorkspaceStats(
+                        projectId, username);
+                mapUserWorkspace.put(username, userStats);
+            }
+        }
+
+        String username = projectDeepMapper.getProjectDetails(projectId)
+                .getOwner().getUserName();
+        if (username == null)
+            return mapUserWorkspace;
+
+        UserStats userStats = (UserStats) setUserWorkspaceStats(projectId,
+                username);
+        mapUserWorkspace.put(username, userStats);
+
+        return mapUserWorkspace;
+    }
+
+    public IUserStats setUserWorkspaceStats(String projectId, String username)

```

Comment:
this is not setting something but returning the requests object, so it shouldn't be called "setUser...". "Set.." implies you're setting a value.